src/foam/nanos/auth/User.js

```diff
@@ -79,8 +79,11 @@ foam.CLASS({
       name: 'birthday'
     },
     {
-      class: 'String',
-      name: 'profilePicture'
+      class: 'Blob',
+      name: 'profilePicture',
+      tableCellFormatter: function (value) {
+        this.tag({ class: 'foam.u2.view.BlobView' });
```

Comment:
You can switch this to ImageBlobView to render the actual image instead of a Download link if you want.

Reply:
Changed